src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -440,6 +445,27 @@ protected function formatterValue(\EntityMetadataWrapper $property_wrapper, \Ent
   }
 
   /**
+   * Gets the list of fields to show in the embedded resource.
+   *
+   * @return string[]
+   *   The list of fields.
+   */
+  protected function emdeddedSparseFieldsets() {
+    // TODO: Inject the request object here.
```

Comment:
The problem is that we don't have the request object in resource fields. Good news is that I (think that I) added support for that in a subsequent PR.